src/App.js

```diff
@@ -1,25 +1,49 @@
-import logo from './logo.svg';
 import './App.css';
+import React, { useState } from 'react';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFood from './components/AddFoodForm';
+import SearchBar from './components/SearchBar';
+import { Row, Col } from 'antd'
 
 function App() {
+  const [allFoods, setAllFoods] = useState(foods)
+  
+  const createFood = (newFood) => {
+    const updatedFood = [newFood, ...allFoods]
+    setAllFoods(updatedFood)
```

Comment:
Every time you create a food you need to update the `allFoods` like you are doing here.
But now since you have another `useState` that will be the result of the search you need to update it as well. 
So under the `setAllFoods(updatedFood)` add this line `setShowFoods(updatedFood)`